adapters/handlers/rest/handlers_backup.go

```diff
@@ -91,9 +91,38 @@ func parseCompressionLevel(l string) ubak.CompressionLevel {
 func (s *backupHandlers) createBackup(params backups.BackupsCreateParams,
 	principal *models.Principal,
 ) middleware.Responder {
+	OverrideBucket := ""
+	OverridePath := ""
+	if params.Body.Config != nil {
+		OverrideBucket = params.Body.Config.Bucket
+		OverridePath = params.Body.Config.Path
+	}
+
+	xAwsAccessKey := ""
+	if params.XAwsAccessKey != nil {
+		xAwsAccessKey = *params.XAwsAccessKey
+	}
+
+	xAwsSecretKey := ""
+	if params.XAwsSecretKey != nil {
+		xAwsSecretKey = *params.XAwsSecretKey
+	}
+
+	xAwsSessionToken := ""
+	if params.XAwsSessionToken != nil {
+		xAwsSessionToken = *params.XAwsSessionToken
+	}
+
 	meta, err := s.manager.Backup(params.HTTPRequest.Context(), principal, &ubak.BackupRequest{
-		ID:          params.Body.ID,
-		Backend:     params.Backend,
+		ID:      params.Body.ID,
+		Backend: params.Backend,
+		Bucket:  OverrideBucket,
+		Path:    OverridePath,
+		Credentials: &backup.Credentials{
```

Comment:
This should be removed, all of the module specific credentials are handled inside the module

Reply:
done